test/test_quantization.py

```diff
@@ -817,15 +817,19 @@ def test_conv(self):
         qconfig_dict = {
             '': default_qconfig
         }
-        model_script = quantize_script(
-            torch.jit.script(conv_model_to_script),
-            qconfig_dict,
-            default_eval_fn,
-            [self.img_data],
-            inplace=False)
-        result_eager = model_eager(self.img_data[0][0])
-        result_script = model_script(self.img_data[0][0])
-        self.assertEqual(result_eager, result_script)
+        for trace in [True, False]:
```

Comment:
Nit: I'd rather trace and script before the loop and then iterate over two scripted models in the loop, i.e.:
```
traced = torch.jit.trace(conv_model_to_script, self.img_data[0][0])
scripted = torch.jit.script(conv_model_to_script)
for model_under_test in [traced, scripted]:
  ...
```
Also, `result_eager` can be computed once before the loop.